src/amule.h

```diff
@@ -117,6 +117,9 @@ namespace Kademlia {
 #define CONNECTED_KAD_FIREWALLED (1<<3)
 
 
+void OnShutdownSignal( int /* sig */ );
```

Comment:
I'd better suppress this warning instead of unnecessarily adding it to the header.

Reply:
what do you mean? what is your suggested changes? keep my previous version and add this line to amule.cpp?

Reply:
Sorry, I got confused with your cppcheck patches. In this case it's okay as-is.